CITATION.cff

```diff
@@ -0,0 +1,20 @@
+# This CITATION.cff file was generated with cffinit.
+# Visit https://bit.ly/cffinit to generate yours today!
+
+cff-version: 1.2.0
+title: cloudmetrics
+message: >-
+  If you use this software, please cite it using the
+  metadata from this file.
+type: software
+authors:
+  - given-names: Leif
+    family-names: Denby
+    email: leif@denby.eu
+    affiliation: University of Leeds
+    orcid: 'https://orcid.org/0000-0002-7611-9222'
+  - given-names: Martin
+    family-names: Janssens
+    email: martin.janssens@wur.nl
+    affiliation: University of Wageningen
```

Comment:
Could we change my affiliation to 'Wageningen University & Research'? The uni is being picky about us using its proper name, I'm sorry...

Reply:
No problem! :)

Reply:
That should be all done now @martinjanssens